util/src/main/java/org/datacommons/util/McfChecker.java

```diff
@@ -67,36 +71,40 @@ public static boolean checkNode(
     Mcf.McfGraph.Builder nodeGraph = Mcf.McfGraph.newBuilder();
     nodeGraph.setType(mcfType);
     nodeGraph.putNodes(nodeId, node);
-    return new McfChecker(nodeGraph.build(), null, null, logCtx).check();
+    return new McfChecker(nodeGraph.build(), null, null, null, logCtx).check();
   }
 
   // Used with Template MCF when there are columns from CSV header.
   public static boolean checkTemplate(
       Mcf.McfGraph graph, Set<String> columns, ExistenceChecker existenceChecker, LogWrapper logCtx)
       throws IOException, InterruptedException {
-    return new McfChecker(graph, columns, existenceChecker, logCtx).check();
+    return new McfChecker(graph, columns, existenceChecker, null, logCtx).check();
   }
 
   private McfChecker(
       Mcf.McfGraph graph,
       Set<String> columns,
       ExistenceChecker existenceChecker,
+      StatVarState svState,
       LogWrapper logCtx) {
     this.graph = graph;
     this.columns = columns;
     this.logCtx = logCtx;
     this.existenceChecker = existenceChecker;
+    this.svState = svState;
   }
 
   // Returns true if there was an sanity error found.
```

Comment:
nit: looking at the logic, it looks like this method returns true if no sanity error is found

Reply:
good catch!